File: back-cidenet/back-cidenet/src/main/java/com/cidenet/prueba/entity/Empleado.java
package com.cidenet.prueba.entity;

import java.util.Date;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Empleado {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String primerApellido;
    private String segundoApellido;
    private String primerNombre;
    private String otroNombre;
    private String pais;
    private String tipoIdentificacion;
    private String numIdentificacion;
    private String correo;
    private Date fechaIngreso;
    private String area;
    private String estado;
    private Date fechaRegistro;

}

File: front-cidenet/front-cidenet/front-cidenet/src/app/models/empleado.ts
export class Empleado {
    id?: number;
    primerApellido :string;
    segundoApellido :string;
    primerNombre :string;
    otroNombre :string;
    pais :string;
    tipoIdentificacion :string;
    numIdentificacion :string;
    correo? : string;
    fechaIngreso: Date;
    area: string;
    estado? : string;
    fechaRegistro? : Date;


    constructor(primerApellido :string,
        segundoApellido :string,
        primerNombre :string,
        otroNombre :string,
        pais :string,
        tipoIdentificacion :string,
        numIdentificacion :string, fechaIngreso: Date,
        area: string) {

        this.primerApellido = primerApellido;
        this.segundoApellido = segundoApellido;
        this.primerNombre = primerNombre;
        this.otroNombre = otroNombre;
        this.pais = pais;
        this.tipoIdentificacion = tipoIdentificacion;
        this.numIdentificacion = numIdentificacion;
        this.fechaIngreso = fechaIngreso;
        this.area = area;
    }
}


